MovingBlocks/Terasology
# Feature: Extraction LightMerger usage in post-ready step.

### Contains

* LightMerger runs after chunk is ready now.
* Simplified ChunkProcessingPipeline.
* Fixed tests
* Faster chunk delivery to ready.. especially for tests! (should not stuck at chunk generation)

### How to test

1. Run tests
2. Run Singleplayer game
2.1. fly on `hspeed`, check how fast chunk loadings.
3. Run Multiplayer
3.1. fly on `hspeed`, check how fast chunk loadings.
